pixel-avatar-hardhat/contracts/PixelAvatars.sol

```diff
@@ -84,17 +63,8 @@ contract PixelAvatars is ERC721Enumerable, ReentrancyGuard, Ownable {
         emit LogTokenMinted(msg.sender, _devDaoTokenId);
     }
 
-    // provide Mint price in wei
-    function setMintPrice(uint256 _newPrice) public onlyOwner {
-        mintPrice = _newPrice;
-    }
-
-    function withdraw() public onlyOwner {
-        payable(msg.sender).transfer(address(this).balance);
-    }
-
     function multiMintWithDevDaoToken(uint256[] memory _devDaoTokenIds)
-        public
+        external
```

Comment:
Why did we change this from `public` from `external`?

Reply:
We're most likely not minting from with-in the contract right? @Dhaiwat10

Reply:
Ah, gotcha